### PR TITLE
Request healthcheck proposals after delay

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

File: modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala [hunk 1]
```diff
@@ -95,21 +95,26 @@ abstract class HealthCheckConsensus[
...
+        } >>
+          missingFromPeers.toList.traverse { id =>
+            requestProposal(id, round.getOwnRoundId).map(_.map((id, _)))
+          }.map(_.flatten)
+            .flatMap(_.traverse {
+              case (peerId, proposal) =>
+                Applicative[F].whenA(peerId === proposal.owner)(handleProposal(proposal))
+            })
+            .void,
+        ???
```
Comment (tbekas) [inline modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala:116]: Missing implementation

Comment (TheMMaciek) [inline modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala:116]: Not implemented false case.

Comment (marcinwadon) [reply modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala]: my bad. fixed

Comment (marcinwadon) [reply modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala]: my bad. fixed

**Review by TheMMaciek — APPROVED** [review]

**Review by TheMMaciek — APPROVED** [review]

**Review by tbekas — APPROVED** [review]

**Pull request merged**